src/cloud-api-adaptor/go.mod

```diff
@@ -47,8 +47,8 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/eks v1.29.5
 	github.com/aws/aws-sdk-go-v2/service/iam v1.22.5
 	github.com/aws/aws-sdk-go-v2/service/s3 v1.38.5
-	github.com/confidential-containers/cloud-api-adaptor/src/cloud-providers v0.9.0-alpha1
-	github.com/confidential-containers/cloud-api-adaptor/src/peerpod-ctrl v0.9.0-alpha1
+	github.com/confidential-containers/cloud-api-adaptor/src/cloud-providers v0.9.0-alpha.1
```

Comment:
do you want to name the tag as alpha.2 ?

Reply:
We could do, but it will restart all the e2e jobs again 😞

Reply:
Oh. leave it..

Reply:
Ah I see you already made the change..